pkg/account/account.go

```diff
@@ -0,0 +1,71 @@
+package account
+
+import (
+	"context"
+
+	"cosmossdk.io/math"
+	"go.uber.org/zap"
+
+	"github.com/archway-network/relayer_exporter/pkg/chain"
+	log "github.com/archway-network/relayer_exporter/pkg/logger"
+)
+
+type Account struct {
+	Address string `yaml:"address"`
+	Denom   string `yaml:"denom"`
+	ChainID string `yaml:"chainId"`
+	Balance math.Int
+}
+
+func GetBalances(accounts []Account, rpcs map[string]string) []Account {
```

Comment:
I think these could be tied to `Account` functions so that there's no need to pass the `Account struct` every time

Reply:
It accepts `[]Account` slice so collections of accounts to get balances for so how to tie it to single `Account`?

Reply:
I mean to have it tied to the struct:
```
func (a *Account) GetBalances() {}
```

Reply:
I think it makes sense for 
```
func (a *Account) GetBalance() {}
```
so GetBalance for single Account

Reply:
In there as well but in general whenever you are passing Structs, consider making them have a `receiver` i.e. `(a *Account)` or `(a Account)`.

Reply:
https://github.com/archway-network/relayer_exporter/pull/12/commits/08469af78832b60a2cb80be2a57c2ced7bcc2b0c